app/assets/stylesheets/components/_background.scss

```diff
@@ -0,0 +1,25 @@
+.tinted-image-dota2 {
+  background:
+    linear-gradient(
+      rgba(4, 5, 45, 0.9),
+      rgba(4, 5, 45, 0.9)
+    ),
+    url(asset_path("dota2_background.png"));
+  height: 100%;
+  background-attachment: fixed;
+  background-repeat: no-repeat;
+  background-size: auto;
+}
+
+.tinted-image-cs {
```

Comment:
you could create a common scss and @extend into the separate scss

e.g. 

common-background {
  height: 100%;
  background-attachment: fixed;
  background-repeat: no-repeat;
  background-size: auto;
}

.tinted-image-cs {
@extend .common-background
    url(asset_path("dota2_background.png"));
}

.tinted-image-dota2 {
@extend .common-background
     url(asset_path("cs_background.jpg"))
}